tests/test_ctx_options.cpp

```diff
@@ -146,6 +146,52 @@ void test_ctx_thread_opts (void *ctx)
 #endif
 }
 
+void test_ctx_zero_copy (void *ctx)
+{
+    int zero_copy;
```

Comment:
@skaes thanks for adding a test, but the body needs a:
```
#ifdef ZMQ_ZERO_COPY_RECV
..
#else
(void)ctx;
#endif
```

as the API is DRAFT, see build failures:
https://travis-ci.org/zeromq/libzmq/jobs/349263452